### ISSUE TITLE
bastion native client

### ISSUE THREAD

**Issue by joehd91** [Azure/azure-cli#23984]
## Describe the bug

**Command Name**
`az network bastion rdp`

**Errors:**
```
The command failed with an unexpected error. Here is the traceback:
cannot import name 'WinDLL' from 'ctypes' (/opt/az/lib/python3.10/ctypes/__init__.py)
Traceback (most recent call last):
  File "/opt/az/lib/python3.10/site-packages/knack/cli.py", line 233, in invoke
    cmd_result = self.invocation.execute(args)
  File "/opt/az/lib/python3.10/site-packages/azure/cli/core/commands/__init__.py", line 663, in execute
    raise ex
  File "/opt/az/lib/python3.10/site-packages/azure/cli/core/commands/__init__.py", line 726, in _run_jobs_serially
    results.append(self._run_job(expanded_arg, cmd_copy))
  File "/opt/az/lib/python3.10/site-packages/azure/cli/core/commands/__init__.py", line 697, in _run_job
    result = cmd_copy(params)
  File "/opt/az/lib/python3.10/site-packages/azure/cli/core/commands/__init__.py", line 333, in __call__
    return self.handler(*args, **kwargs)
  File "/opt/az/lib/python3.10/site-packages/azure/cli/core/commands/command_operation.py", line 121, in handler
    return op(**command_args)
  File "/opt/az/lib/python3.10/site-packages/azure/cli/command_modules/network/custom.py", line 8396, in rdp_bastion_host
    from ._process_helper import launch_and_wait
  File "/opt/az/lib/python3.10/site-packages/azure/cli/command_modules/network/_process_helper.py", line 9, in <module>
    from ctypes import WinDLL, c_int, c_size_t, Structure, WinError, sizeof, pointer
ImportError: cannot import name 'WinDLL' from 'ctypes' (/opt/az/lib/python3.10/ctypes/__init__.py)

```

## To Reproduce:
Steps to reproduce the behavior. Note that argument values have been redacted, as they may contain sensitive information.

- _Put any pre-requisite steps here..._
- `az network bastion rdp --name {} --resource-group {} --target-resource-id {}`

## Environment Summary
```
Linux-5.10.102.1-microsoft-standard-WSL2-x86_64-with-glibc2.31, Ubuntu 20.04.5 LTS
Python 3.10.5
Installer: DEB

azure-cli 2.40.0
```

**Comment by navba-MSFT** [comment]
@joehd91 Thanks for reaching out to us and reporting this issue. We are looking into this issue and we will provide an update.

**Comment by yonzhan** [comment]
route to CXP team

**Comment by navba-MSFT** [comment]
@joehd91 Could you please confirm if you are running this from a cloudshell on Linux ? If Yes, it wont be able to open the RDP on your local machine. For this reason, we recommend using powershell with az cli (on Windows machine).

Also, note that RDP command works best on Windows, for other OS’ (like Linux) you can do the [tunnel command](https://docs.microsoft.com/en-us/azure/bastion/connect-native-client-windows#connect-tunnel) and then use the client of your choice.

>Connect to a VM using the [az network bastion tunnel command](https://docs.microsoft.com/en-us/azure/bastion/connect-native-client-windows#connect-tunnel). This lets you do the following:
>- Use native clients on non-Windows local computers (example: a Linux PC).
>- Use the native client of your choice. (This includes the Windows native client.)
>- Connect using SSH or RDP.
>- Set up concurrent VM sessions with Bastion.
>- [Upload files](https://docs.microsoft.com/en-us/azure/bastion/vm-upload-download-native#tunnel-command) to your target VM from your local computer. File download from the target VM to the local client is currently not supported for this command.

>Currently, this feature has the following limitation:
>
>Signing in using an SSH private key stored in Azure Key Vault isn’t supported with this feature. Before signing in to your Linux VM using an SSH key pair, download your private key to a file on your local machine.

Hope this helps.

**Comment by navba-MSFT** [comment]
@joehd91 I wanted to do quick follow-up to check if you had a chance to look at my above comment. Please let us know if you had any updates on this. Awaiting your reply.

**Comment by navba-MSFT** [comment]
@joehd91 The action is currently pending on you to try the above suggestion. If you need any further assistance on this issue in future, please feel free to reopen this thread. We would be happy to help.

**Comment by jiasli** [comment]
Duplicate of #23143
